softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/ActionHeaderFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.softgroup.common.protocol;
+
+import java.util.UUID;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 30.03.17.
+ */
+public class ActionHeaderFactory {
+    public static ActionHeader createHeader(ActionHeader requestHeader){
+        return ActionHeader.newBuilder()
+                .setType(requestHeader.getType())
+                .setCommand(requestHeader.getCommand())
+                .setVersion(requestHeader.getVersion())
+                .setOriginUuid(requestHeader.getUuid())
+                .setUuid(UUID.randomUUID().toString())
```

Comment:
@Dominys response header UUID must be only one and the same for each response message ?